app/views/content/fund-your-training/scholarships-and-bursaries.md

```diff
@@ -0,0 +1,65 @@
+
+Postgraduate teaching bursaries and scholarships are only available for the subjects listed below. You cannot receive both a teaching bursary and a scholarship.
+
+| Subject                                          | Bursary  | Scholarship      |
+| -------------                                    | -------: | ---------------: |
+| **Biology**                                      | £10,000  |                  |
+| **Chemistry**                                    | £24,000  | £26,000          |
+| **Computing**                                    | £24,000  | £26,000          |
+| **Design and technology**                        | £15,000  |                  |
+| **Geography**                                    | £15,000  |                  |
+| **Languages**<br/> (including ancient languages) | £15,000  |                  |
+| **Maths**                                        | £24,000  | £26,000          |
+| **Physics**                                      | £24,000  | £26,000          |
+
+ To be eligible for a bursary, you’ll need at least one of the following:
+
+- a bachelor’s degree class 2:2 (honours) or higher
+- a master’s degree
+- a PhD
+
+You’ll also need to meet the bursary scheme’s individual terms and conditions.
+
+[Find out more about your eligibility for teaching bursaries](https://www.gov.uk/government/publications/funding-initial-teacher-training-itt/funding-initial-teacher-training-itt-academic-year-2022-to-2023)
+
+To find out if you’re eligible for a scholarship and to apply, you can visit the relevant scholarship body:
+
+- [the Royal Society of Chemistry](https://www.rsc.org/awards-funding/funding/teacher-training-scholarships/) (chemistry)
+- [BCS The Chartered Institute for IT](https://www.bcs.org/get-qualified/certification-and-scholarships-for-teachers/bcs-computer-teacher-scholarships/) (computing)
+- [the Institute of Mathematics and its Applications](http://teachingmathsscholars.org/about) (maths)
+- [the Institute of Physics](https://www.iop.org/about/support-grants/iop-teacher-training-scholarships#gref) (physics)
+
+## Undergraduate bursaries
+
+You may be eligible for a [bursary of £9,000](https://www.gov.uk/government/publications/funding-initial-teacher-training-itt/funding-initial-teacher-training-itt-academic-year-2022-to-2023#undergraduate-bursary) if you do a secondary maths or physics course that leads to [qualified teacher status](/what-is-qts) (QTS).
+
+You may also be eligible if you do an opt-in QTS course in secondary computing, languages, mathematics or physics. This is where you start a standard undergraduate degree but then opt-in to initial teacher training to get QTS.
+
+The amount you receive depends on the year that you start your QTS course. You do not get a bursary in every year of an undergraduate course. Depending on the type of course you do, you may get the bursary in your final year or final two years.
+
+### If you are former military personnel
```

Comment:
This h3 heading was flagged in the DAC accessibility report - they thought it should be an h2

However, I think it is right as an h3 as this is an undergrad bursary

If there a way of renaming it @EllieNodder so it is less confusing?